src/directives/decorators/bootstrap/formselect.html

```diff
@@ -0,0 +1,10 @@
+<div sf-form-select="form" class="schema-form-form-select {{form.htmlClass}}">
```

Comment:
This has now moved to https://github.com/json-schema-form/angular-schema-form-bootstrap

Reply:
We should probably create a simple one for core.